src/main/resources/META-INF/plugin.xml

```diff
@@ -62,6 +62,12 @@
             <action id="com.redhat.devtools.intellij.knative.actions.func.DeployAction"
                     class="com.redhat.devtools.intellij.knative.actions.func.DeployAction"
                     text="Deploy"/>
+            <action id="com.redhat.devtools.intellij.knative.actions.func.AddEnvAction"
+                    class="com.redhat.devtools.intellij.knative.actions.func.AddEnvAction"
+                    text="Add Envs"/>
```

Comment:
I think we should reword it into a more user friendly: Add environment info or Add environment variable ?